docs/upgrade.html

```diff
@@ -19,6 +19,16 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
+<h4><a id="upgrade_3_5_0" href="#upgrade_3_5_0">Upgrading to 3.5.0 from any version 0.8.x through 3.4.x</a></h4>
+
+    <h5><a id="upgrade_350_notable" href="#upgrade_350_notable">Notable changes in 3.5.0</a></h5>
+    <ul>
+        <li>The JmxTool has been migrated to the tools module.
```

Comment:
For curiosity, do you envision for us to add a similar `<li>` here for each tool as we move them across?

Reply:
No, only tools that do not have a wrapper script (e.g. JmxTool and StateChangeLogMerger).